<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Suppliers</class>
 <widget class="QMainWindow" name="Suppliers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>270</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Поставщики</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/Мастер пол.ico</normaloff>../../Downloads/Мастер пол.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#123456;
padding: 0;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableSuppliers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>501</width>
      <height>171</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
max-width: 700px;
background-color:#666000;</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>№</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ФИО</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>История</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тип</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ИНН</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="findSuppliers">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>&quot;Segoe UI Black&quot;</family>
      <pointsize>10</pointsize>
      <weight>10</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;
max-width: 700px;
background-color:#white;

color: black;
font: 87 10pt \&quot;Segoe UI Black\&quot;;\n</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string> Поиск</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addNewSupplierButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#666000;

border: 6px;
color: black;
font: 87 10pt \&quot;Segoe UI Black\&quot;;\n</string>
    </property>
    <property name="text">
     <string>Добавить нового поставщика</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
